index.js

```diff
@@ -319,19 +321,15 @@ module.exports = {
     }
 
     function listMembers(groupId, opts = {}) {
+      const { live } = opts
+
       return pull(
-        getPickedEpoch(groupId, { live: !!opts?.live }),
-        pull.map((pickedEpoch) => {
-          console.log('pickedEPoch', pickedEpoch)
-          return getMemberUpdates(pickedEpoch.id, { live: !!opts?.live })
-        }),
+        getPreferredEpoch.stream(groupId, { live }),
+        pull.map((epoch) => getMembers.stream(epoch.id, { live })),
         pull.flatten(),
-        pull.map((update) => {
-          if (update.added) return update.added
-          return update
-        }),
+
         // if it's not live, only include adds
-        pull.filter((update) => opts?.live || !update.excluded)
+        live ? null : pull.filter((update) => update.added)
```

Comment:
not sure why i added this line to start with, if it's not live then it's not going to output any exclusions anyway (if someone's excluded then we'd switch to a new epoch without exclusions on it)